lms/static/js/Markdown.Converter.js

```diff
@@ -551,24 +552,27 @@ else
         }
 
         function writeAnchorTag(wholeMatch, m1, m2, m3, m4, m5, m6, m7) {
-            if (m7 == undefined) m7 = '';
+            var result;

```

Comment:
what's with m7 removal?

Reply:
it was moved below. eslint was previously failing on this as variables must be declared at the top of the file.